x-pack/packages/observability/alert_details/src/components/alert_time_range_annotation.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { RectAnnotation } from '@elastic/charts';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  alertStart: number;
+  alertEnd?: number;
+  color: string;
+  id: string;
+}
+
+const RECT_ANNOTATION_TITLE = i18n.translate(
+  'observabilityAlertDetails.alertTimeRangeAnnotation.detailsTooltip',
+  {
+    defaultMessage: 'Active',
+  }
+);
+
+export function AlertTimeRangeAnnotation({ alertStart, alertEnd, color, id }: Props) {
+  return (
+    <RectAnnotation
+      id={id}
+      dataValues={[
+        {
+          coordinates: {
+            y0: 0,
+            x0: alertStart,
+            x1: alertEnd,
+          },
+          details: RECT_ANNOTATION_TITLE,
+        },
+      ]}
+      style={{ fill: color, opacity: 0.1 }}
```

Comment:
I think we need to increase the opacity; it's not very visible Active shade vs Threshold shade 
<img width="529" alt="Screenshot 2023-04-05 at 18 09 28" src="https://user-images.githubusercontent.com/6838659/230139978-059474a8-e350-495f-96ee-40424a83944d.png">

Reply:
I used the opacity that was mentioned in Figma, let's double check this one with @maciejforcone too.